adapters/ix/ix.go

```diff
@@ -417,6 +420,28 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 				}
 			}
 
+			var bidNative1_1 *Native11Wrapper
+			if bidType == openrtb_ext.BidTypeNative {
+				unmarshalExtErr := json.Unmarshal([]byte(bid.AdM), &bidNative1_1)
```

Comment:
This variable name could be better since you aren't unmarshallng an ext field. Perhaps `unmarshalErr` or `err` would work.

Reply:
All set, renamed to `unmarshalErr` and `marshallErr` to keep it simple